app/helpers/injection_helper.rb

```diff
@@ -10,12 +10,8 @@ def inject_enterprises(enterprises = Enterprise.activated.includes(address: :sta
     )
   end
 
-  def inject_map_pointers
-    inject_json_ams(
-      'enterprises',
-      Enterprise.activated.includes(address: :state).all,
-      Api::ShopsSerializer
-    )
+  def inject_shopfronts
+    inject_json_ams 'enterprises', Enterprise.activated.includes(address: :state).all, Api::EnterpriseShopfrontListSerializer
```

Comment:
this is now failing on codeclimate, line too long... some break lines will do
btw, this is should now be  def inject_enterprise_shopfront_list